test/Common/TestActivities.cs

```diff
@@ -100,6 +100,11 @@ public static double BindToDouble([ActivityTrigger] double value)
             return httpManagementPayload;
         }
 
+        public static Task Delay([ActivityTrigger] TimeSpan duration)
```

Comment:
Is this being used somewhere?

Reply:
Not anymore. I created it to test something. I can remove it.